__tests__/api.test.js

```diff
@@ -308,7 +306,91 @@ const pageCount = [
     })
   }),
 
+  describe('POST /api/articles/:article_id/comments', () => {
+    describe('successful connection test(s)', () => {
+      const comment1 = {
+        body: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit',
+        author: 'lurker',
+      }
+      test('201: page returns with an object', () => {
+        return request(app)
+          .post('/api/articles/1/comments')
+          .send(comment1)
+          .expect(201)
+          .then(({
+            body
+          }) => {
+            expect(typeof body).toBe('object')
+          })
+      })
+      test('201: returns posted comment', () => {
 
+        return request(app)
+          .post('/api/articles/1/comments')
+          .send(comment1)
+          .expect(201)
+          .then(({
+            body
+          }) => {
+            expect(body).toHaveProperty("comment_id", expect.any(Number))
+            expect(body).toHaveProperty("body", expect.any(String))
+            expect(body).toHaveProperty("article_id", expect.any(Number))
```

Comment:
I would check that the article_id is specifically 1 as you've specified it in your path